CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
 This file lists the main changes with each version of the Fyne toolkit.
 More detailed release notes can be found on the [releases page](https://github.com/fyne-io/fyne/releases). 
 
-## 2.4.1 - 8 October 2023
```

Comment:
I think you may have based your branch off of master instead of develop, or otherwise accidentally picked up this unrelated diff?

Reply:
Yeah. It was based on master. Sorry! I will close this one and create a new pull request.